modules/post/windows/gather/checkvm.rb

```diff
@@ -44,6 +47,14 @@ def service_exists?(service)
     get_services && get_services.include?(service)
   end
 
+  def get_computer_model
+    wmic_query('computersystem get model')
+  end
```

Comment:
```suggestion
  def get_computer_model
    @get_computer_model ||= wmic_query('computersystem get model')
  end
```
This prevents us from executing the command multiple times- same for the other data retrieval methods below.